util.js

```diff
@@ -33,8 +33,44 @@ debugerror.log = console.error.bind(console)
 /** @function */
 const isBinaryFileAsync = module.exports.isBinaryFileAsync = Promise.promisify(require('isbinaryfile'))
 
+function removePassword (input) {
+  return input.replace(/(-P |pass:|\/p|-pass )([^ ]+)/, function (match, p1, p2) {
+    return `${p1}***`
+  })
+}
+
 /** @function */
-module.exports.execFileAsync = Promise.promisify(child.execFile)
+module.exports.execFileAsync = function (file, args, options) {
+  if (debug.enabled) {
+    debug(`Executing ${file} ${args == null ? '' : removePassword(args.join(' '))}`)
+  }
+
+  return new Promise((resolve, reject) => {
```

Comment:
@develar would you help remove the Node support in in [`.travis.yml`](https://github.com/electron-userland/electron-osx-sign/blob/master/.travis.yml) as well if we're dropping `0.12` and some others? Travis didn't pick up the error because it doesn't run through the actual code but a quick syntax/consistency check.